packages/vite-node/src/cli.ts

```diff
@@ -1,71 +1,46 @@
-import minimist from 'minimist'
+import cac from 'cac'
 import { dim, red } from 'kolorist'
 import { createServer } from 'vite'
+import { version } from '../package.json'
 import { ViteNodeServer } from './server'
 import { ViteNodeRunner } from './client'
 
-const argv = minimist(process.argv.slice(2), {
-  'alias': {
-    r: 'root',
-    c: 'config',
-    h: 'help',
-    w: 'watch',
-    s: 'silent',
-  },
-  '--': true,
-  'string': ['root', 'config'],
-  'boolean': ['help', 'watch', 'silent'],
-  unknown(name: string) {
-    if (name[0] === '-') {
-      console.error(red(`Unknown argument: ${name}`))
-      help()
-      process.exit(1)
-    }
-    return true
-  },
-})
-
-if (argv.help) {
-  help()
-  process.exit(0)
-}
-
-if (!argv._.length) {
-  console.error(red('No files specified.'))
-  help()
-  process.exit(1)
-}
+const cli = cac('vite-node')
 
-// forward argv
-process.argv = [...process.argv.slice(0, 2), ...(argv['--'] || [])]
+cli
+  .version(version)
+  .option('-r, --root <path>', 'Use specified root directory')
+  .option('-c, --config <path>', 'Use specified config file')
+  .option('-w, --watch', 'Restart on file changes, similar to "nodemon"')
+  .option('-s, --silent', 'Do not emit errors and logs')
+  .option('--vue', 'Support for importing Vue component')
+  .help()
 
-run(argv)
+cli
+  .command('[...files]')
+  .action(run)
 
-function help() {
-  // eslint-disable-next-line no-console
-  console.log(`
-Usage:
-  $ vite-node [options] [files]
-
-Options:
-  -r, --root <path>      ${dim('[string]')} use specified root directory
-  -c, --config <file>    ${dim('[string]')} use specified config file
-  -w, --watch           ${dim('[boolean]')} restart on file changes, similar to "nodemon"
-  -s, --silent          ${dim('[boolean]')} do not emit errors and logs
-  --vue                 ${dim('[boolean]')} support for importing Vue component
-`)
-}
+cli.parse()
 
 export interface CliOptions {
-  files?: string[]
-  _?: string[]
   root?: string
   config?: string
   watch?: boolean
+  // silent?: boolean
+  // vue?: boolean
```

Comment:
Are those two options still necessary? I could not find any reference to them. Were they forgotten in a previous refactor?